packages/components/transfer/src/transfer.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+  <div :class="ns.b()">
+    <transfer-panel
+      ref="leftPanel"
+      :data="sourceData"
+      :option-render="optionRender"
+      :placeholder="panelFilterPlaceholder"
+      :title="leftPanelTitle"
+      :filterable="filterable"
+      :format="format"
+      :filter-method="filterMethod"
+      :default-checked="leftDefaultChecked"
+      :props="props.props"
```

Comment:
this can also be removed

Reply:
I think we can't just remove `props` here, but should define `propsAlias` in `transfer.vue` and pass it to `transfer-panel.vue`, like:

```ts
const propsAlias = usePropsAlias(props)
```
and,

```vue
<transfer-panel
  ...
  :props="propsAlias"
>
  ...
</transfer-panel>
```

Reply:
Hi, @tolking what's your idea?🤔 waiting for your reply.

Reply:
You can use the `usePropsAlias` functions directly in `transfer-panel` component, why pass it?

Reply:
because `usePropsAlias` needs the `props` of `transfer` as input parameters.

Reply:
you are right, i forgot about this.